client/my-sites/upgrades/checkout/supporting-text.jsx

```diff
@@ -6,6 +6,7 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var cartValues = require( 'lib/cart-values' ),
+	support = require( 'lib/support' ),
```

Comment:
@drewblaisdell The constants are generated for usages throughout Calypso, but I've changed the "usages" for upgrades (otherwise PR was going to be very big).

The reason I didn't want to use `paths` module is (afaik) that module is used for paths _inside_ Calypso, whereas these links are outside links.